backend/app/services/ai/chat_service_enhanced.py

```
@@ -0,0 +1,332 @@
+
+集成新的 JSONL 解析器，提供更强大的内容处理能力：
+- 高性能 JSONL 解析
+- 智能错误恢复
+- 详细的错误报告
+- 缓存和优化
+- 多格式输出支持
+"""
+
+import json
+import logging
+from typing import Any, Dict, List, Optional, Union
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)